gluecodium/src/main/resources/templates/cpp/Locale.mustache

```diff
@@ -0,0 +1,89 @@
+{{!!
+  !
+  ! Copyright (C) 2016-2020 HERE Europe B.V.
+  !
+  ! Licensed under the Apache License, Version 2.0 (the "License");
+  ! you may not use this file except in compliance with the License.
+  ! You may obtain a copy of the License at
+  !
+  !     http://www.apache.org/licenses/LICENSE-2.0
+  !
+  ! Unless required by applicable law or agreed to in writing, software
+  ! distributed under the License is distributed on an "AS IS" BASIS,
+  ! WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ! See the License for the specific language governing permissions and
+  ! limitations under the License.
+  !
+  ! SPDX-License-Identifier: Apache-2.0
+  ! License-Filename: LICENSE
+  !
+  !}}
+// -------------------------------------------------------------------------------------------------
+// Copyright (C) 2016-2020 HERE Europe B.V.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+// SPDX-License-Identifier: Apache-2.0
+// License-Filename: LICENSE
+//
+// -------------------------------------------------------------------------------------------------
+
+#pragma once
+
+#include "Optional.h"
+#include <string>
+
+{{#internalNamespace}}
+namespace {{.}} {
+{{/internalNamespace}}
+
+/**
+ * Represents an ISO locale, optionally with a corresponding BCP 47 language tag.
+ */
+struct Locale {
+    Locale();
+    explicit Locale(const optional<std::string> language_tag);
+    Locale(const optional<std::string> language_code, const optional<std::string> country_code);
+    Locale(const optional<std::string> language_code,
```

Comment:
const is not movable :(

Reply:
Curiously, it's not even a compiler warning. Don't we all love C++, such a modern language it is.